scripts/build-homepage

```diff
@@ -2,18 +2,34 @@
 # This script is sourced on build server, shebang is for convenience
 # bash is used to have a predictable ulimit behavior
 
+set -e
```

Comment:
When we source this file from jenkins, a functionality similar to set -ex is already present.

Reply:
For testing purposes it's very helpful. Additionally it helps the reader to understand the script a little bit better.

Reply:
Yes, without a doubt it is useful for standalone execution. Executed from within jenkins, however, `-x` might print every command twice? (`-e` should not cause any troubles, though).